pkgs/by-name/ti/tidgi/package.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tidgi";
+  version = "0.9.6";
+
+  src =
+    if stdenv.isAarch64 then
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-arm64-${version}.zip";
+          hash = "sha256-1Z9lxZZWrUVQEhBO/Kt2AS/uNs2XfihdL0iGrguPQ5g";
+        }
+    else
+      fetchurl
+        {
+          url = "https://github.com/tiddly-gittly/TidGi-Desktop/releases/download/v${version}/TidGi-darwin-x64-${version}.zip";
+          sha256 = "sha256-1mbj9dchgjmnfvrjh56ql99xbsq6n7kdr0fhww3j7rd4x63afjnm";
```

Comment:
Unfortunately eval fails due to this not being a proper SRI sha-256 hash. Remember to get the hash using `nix-prefetch-url` and convert it to SRI hash properly.